longhorn/longhorn.go

```diff
@@ -0,0 +1,24 @@
+package longhorn
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// GetVolumeNameFromReplicaName returns the volume name from the replica name.
```

Comment:
Maybe it's worth mentioning that the replica name is the data directory name, not the replica CR object name (with -r-).  Might even be worth a test case for that.